src/main/java/school/lemon/changerequest/java/introduction/hw1/Task6.java

```diff
@@ -2,10 +2,16 @@
 
 public class Task6 {
     public static double calculateS(double x) {
-        return (1 + (x * (((Math.pow(x, 2) *x) *x) / (2 * (3*2) * (4*3*2)))));
+        return (1 + (x * (((Math.pow(x, 2) *x) *x) / (factorial(2)*factorial(3)*factorial(4)))));
```

Comment:
Your task is to calculate: `1 + x + x^2/2! + x^3/3! + x^4/4!`
And you are trying to calculate: `1 + (x * x^2 * x * x) / (2! * 3! * 4!)`